js/data/bucket/symbol_bucket.js

```diff
@@ -37,6 +37,11 @@ function SymbolBucket(options) {
     this.adjustedTextSize = options.adjustedTextSize;
     this.adjustedIconSize = options.adjustedIconSize;
     this.fontstack = options.fontstack;
+
+    // this constant is based on the size of the glyphQuadEndIndex and iconQuadEndIndex
+    // in the symbol_instances StructArrayType
+    // eg the max valid UInt16 is 65,535
+    this.MAX_QUADS = 65535;
```

Comment:
Making `MAX_QUADS` a static property instead of a prototype property on `SymbolBucket` would provide more accurate contextual clues about what it is and how it works.